<!-- -->
<launch>

  <node pkg="driverless"  type="lidar_node" name="lidar_node">
    <param name="dev_path"         type="string" 	value="/dev/ttyUSB0"/>
    <param name="frame_id"            type="string" value="/lidar_frame"/>
    <param name="scan_topic"            type="string"   value="/scan"/>
  </node>
  
  <node name="lower_control_node" pkg="driverless" type="lower_control_node">

    <param name="port_name" value="/dev/ttyUSB1" />
    <param name="odom_frame" value="odom" />
    <param name="base_frame" value="base_link"/>

    <param name="baud_rate" value="115200" />
    <param name="control_rate" value="10" />
    <param name="sensor_rate" value="5" />

    <param name="reduction_ratio" value="1.0" />
    <param name="encoder_resolution" value="1600.0" />

    <param name="wheel_diameter" value="0.25" />
    <param name="model_param" value="0.83" />
    <param name="pid_rate" value="50.0" />
	<param name="maximum_encoding" value="32.0" />
 </node>
 
 <node name="upper_control_node" pkg="driverless" type="upper_control2_node" output="screen">
	<param name="linear_speed" value="0.1"/>
	<param name="file_path" value="/home/ubuntu/projects/catkin_ws/src/driverless/data/1.txt"/>
 </node>
 
 
 <node name="rviz" pkg="rviz" type="rviz"/>

</launch>
